examples/networking/xdp/xdp_drop_count.py

```diff
@@ -0,0 +1,123 @@
+from bcc import BPF
+import time
+import sys
+
+
```

Comment:
Can you please remove the whitespace?

Also, it would be good to add your own copyright notice to the top of the file.